includes/definitions/teradici-pcoip.yaml

```diff
@@ -0,0 +1,11 @@
+os: teradici-pcoip
+text: 'Teradici PCoIP'
+type: workstation
```

Comment:
For now that line isn't used but it's best to keep it until we either add support or remove those config lines.